### DISCUSSION TITLE
Please make it free

### DISCUSSION THREAD

**Discussion by SagheerHussainZardari-PayPro** [community/community#19608] [Copilot Conversations]
Please make Copilot  free

**Comment by melvincwng** [comment]
@SagheerHussainZardari-PayPro, may I ask, are you a) a current student or b) a maintainer of popular open-source projects?

If you are either of the two, GitHub Copilot would be free for you to use.

For additional information, you can refer to [this link](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/) for more info.

Hope this clarifies ^^.

**Reply by ollie-iterators** [reply]
I think that there should be a way to make it cheaper/free for people who have well maintained code which could make a virtuous loop of improvement for Copilot.

**Reply by melvincwng** [reply]
@ollie-iterators Hi Ollie, I totally agree with you. 👍🏻 

But unfortunately, there's nothing much we can do with regards to GitHub Copilot pricing. Other than discussing it over here, and hopefully one of the staff might be able to pick up the suggestions posted by the community, and take them into consideration.

I've been also seeing quite a number of posts on making GitHub Copilot free for educators - this I fully agree with as well. 

After all, our educators work tirelessly to educate the future generation of developers; would be great if they can use co-pilot for free as well, alongside their students (for context purposes - verified students get to use CoPilot for free).

**Comment by Zoltus** [comment]
@SagheerHussainZardari-PayPro Totally agreed! I grateful its free for students, I couldnt afford it otherwise.

**Comment by lassejlv** [comment]
How do i get copilot free if im a student

**Reply by Zoltus** [reply]
Here are the instructions. All students get the githubpilot for free.
https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack

**Comment by MrPotatoXx** [comment]
Apparently if you have a student account you can still use githubcopilot for free

**Reply by lassejlv** [reply]
Okay thanks alot!

**Comment by ollie-iterators** [comment]
I made a poll about making GitHub Copilot free.

**Reply by lassejlv** [reply]
> I made a poll about making GitHub Copilot free.

Ohhh cool, where ?

**Reply by martin-braun** [reply]
https://github.com/community/community/discussions/20284
